### PR TITLE
feat: add "About the App" screen

### PR DESCRIPTION
**Pull request by jonowu**
## Purpose

Credit the creators of the app

## Changes

Add "About the App" screen in profile.

### Screenshots

Screen Shot 2020-10-25 at 12 06 56 pm
Screen Shot 2020-10-25 at 12 06 59 pm

## Checklist

- [ ] Tested on Android
- [ ] Tested on iOS

### CODE REVIEW

**Comment by jonowu** [comment]
I will run `expo publish` when this is merged, this will make any recent updates live.

File: screens/about/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+const StyledBody = styled(Body)`
+  text-align: center;
+`;
+
+const About = () => {
+  return (
+    <ChildScreen heading="About the App">
+      <StyledBody variant={4}>
+        {`The Swinburne Actions for Sustainability (SA4SI) app was developed in 12 weeks as part of a capstone industry project unit.\n\nDeveloped by: Jonathan Wu, Benjamin Yong, Ian Song\n\nDesigned by: Taïs Lildaree`}
```
Comment (BenjaminYong) [inline screens/about/index.js:15]: - Swinburne Actions for Sustainability Impact*
- developed over* 12 weeks

Maybe a paragraph before we list our names along the lines of this:

'The SA4SI initiative aims to both profile sustainable actions undertaken by the Swinburne community and encourage action on the United Nations Sustainable Development Goals (SDGs).'

Comment (jonowu) [reply screens/about/index.js]: Fixed

**Review by BenjaminYong — APPROVED** [review]
Sweet, looks good!
Thanks mate 😄 

Check out my PR fix when you're free: https://github.com/jonowu/sustainability-app-demo/pull/37

**Pull request merged**